rules/windows/persistence_user_account_added_to_privileged_group_ad.toml

```diff
@@ -12,7 +12,7 @@ any action in Active Directory and on domain-joined systems.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
```

Comment:
endpoint populate the iam data ?

Reply:
definitely not - should remove the index 👍

Reply:
removed